nixos/modules/hardware/opengl.nix

```diff
@@ -12,10 +12,7 @@ let
 
   makePackage = p: pkgs.buildEnv {
     name = "mesa-drivers+txc-${p.mesa.version}";
-    paths =
-      [ p.mesa.drivers
-        (if cfg.s3tcSupport then p.libtxc_dxtn else p.libtxc_dxtn_s2tc)
-      ];
+    paths = [ p.mesa.drivers ];
```

Comment:
I see no need `buildEnv` from a single package.  Perhaps `makePackage = p: p.mesa.drivers` or even scrap the function completely.

Reply:
Done.